app/translations/messages+intl-icu.en.xlf

```diff
@@ -74,6 +74,14 @@
                 <source>error.common.invalid.json</source>
                 <target>Invalid JSON</target>
             </trans-unit>
+            <trans-unit id="sucess_ldap">
+                <source>sucess.ldap</source>
+                <target>Sucess</target>
+            </trans-unit>
+            <trans-unit id="error_ldap_deserialize">
+                <source>error.ldap.deserialize</source>
+                <target>The LDAP entry it's not in a good format</target>
+            </trans-unit>
```

Comment:
Fix typo, also group translation

also... given the "success" message being extremely generic, not sure it needs to be tagged `ldap`. and as discussed yesterday, the current use case of the success message is not relevant.
```suggestion
            <trans-unit id="error_ldap_deserialize">
                <source>error.ldap.deserialize</source>
                <target>The LDAP entry it's not in a good format</target>
            </trans-unit>
            <trans-unit id="success_common_message">
                <source>success.common.message</source>
                <target>Success</target>
            </trans-unit>
```